tree/dataframe/src/RLoopManager.cxx

```diff
@@ -688,7 +688,15 @@ std::vector<std::string> RLoopManager::GetFiltersNames()
    return filters;
 }
 
-std::vector<RDFInternal::RActionBase *> RLoopManager::GetAllActions()
+std::vector<RNodeBase *> RLoopManager::GetGraphEdges() const
+{
+   std::vector<RNodeBase *> nodes;
+   nodes.insert(nodes.end(), fBookedFilters.begin(), fBookedFilters.end());
```

Comment:
Could `std::vector<RNodeBase *> nodes(fBookedFilters.begin(), fBookedFilters.end());` work?

Reply:
refactored to remove the extra allocations